module/api/api.js

```diff
@@ -48,92 +49,130 @@ Object.prototype.exists = function(property_name, value) {
 
 // Check if value exists in array
 Array.prototype.exists = function(value) {
-     for (let i = 0; i < this.length; i++)
```

Comment:
Why not use [`Array.prototype.includes`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)?